ghc.nix

```diff
@@ -222,6 +222,7 @@ hspkgs.shellFor rec {
 
     ${lib.optionalString withDocs "export FONTCONFIG_FILE=${fonts}"}
 
+    # N.B. This overrides CC, CONFIGURE_ARGS, etc. to configure the cross-compiler.
```

Comment:
I believe that this “N.B.” could be copied in the README! AFAIU, it could be surprising to just add `withWasiSDK = true;` is a `shell.nix` and don't realize that now `hadrian/build-cabal` builds a WASM cross-compiler _by default_ …

Reply:
Maybe (in another PR) it would worth renaming it to `useWasiSDK = true;` since it has a semantic more similar to, e.g., `useClang = true;` rather than, e.g., `withEMSDK = true;`, WDYT?

Reply:
I don't have a strong opinion for either; as long as it's documented that the `withWasiSDK`/`useWasiSDK` flag makes it default to building the wasm backend